client/src/App.scss

```diff
@@ -205,55 +179,43 @@ Item overview
 }
 
 .item-card {
-  padding: 0 (2.5 * $unit) 0 0;
+  padding: 0 $unit 0 0;
   background-color: $black100;
   border-bottom: $border;
-}
-.item-card:hover {
-  cursor: pointer;
+  box-shadow: $shadow-medium;
+  display: grid;
+  grid-template-columns: 9fr 1fr;
+
+  &.expanded {
+    box-shadow: none;
+  }
 }
 
 .item-card__clickable {
-  padding: (2.5 * $unit) 0 (2.5 * $unit) (2.5 * $unit);
-  width: 100%;
+  padding: (2 * $unit) 0 (2 * $unit) (2 * $unit);
+  width: 95%;
+  display: grid;
+  grid-template-columns: 8fr 6fr 5fr 4fr;
+}
+
+.item-card__clickable:hover {
+  cursor: pointer;
```

Comment:
does the Touchable not already do `cursor: pointer`?

Reply:
yes it does!